protocol/src/main/java/org/fido/iot/protocol/epid/EpidHttpClient.java

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Intel Corporation
+// SPDX-License-Identifier: Apache 2.0
+
+package org.fido.iot.protocol.epid;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.time.Duration;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class EpidHttpClient {
+
+  private static ExecutorService executor = Executors.newCachedThreadPool(r -> {
+    Thread t = Executors.defaultThreadFactory().newThread(r);
+    t.setDaemon(true);
+    return t;
+  });
+  private static final long httpRequestTimeout = Duration.ofSeconds(10).getSeconds();
+
+  /**
+   * Perform HTTP GET operation.
+   * 
+   * @param url the Url to which request will be sent
+   * @return the response from the Url
+   * @throws IOException throws an IOException in case the response status code is not 200 (OK)
+   */
+  public static byte[] doGet(String url) throws IOException {
+    HttpClient httpClient;
+    try {
+      httpClient = HttpClient.newBuilder().build();
+      final HttpRequest.Builder httpRequestBuilder =
+          HttpRequest.newBuilder().header("Accept", "application/octet-stream");
```

Comment:
Should be "application/cbor"?

Reply:
The REST call is being made to EPID online Verifier Service which is not being migrated to use CBOR, since it lies outside of the protocol's scope.